docs/openapi.yml

```diff
@@ -370,6 +404,17 @@ components:
         maxLength: 1024
         example: urn:datasource:postgresql:financials_db
 
+    datasetUrn:
+      name: urn
+      in: path
+      description: The URN of the dataset.
+      required: true
+      schema:
+        type: string
+        format: URN
+        maxLength: 1024
+        example: urn:datasource:postgresql:financials_db
```

Comment:
This looks more like a datasource urn?

Reply:
Good catch! Getting a bit lazy with my copy/paste :)